apps/vmq_server/src/vmq_queue.erl

```diff
@@ -934,6 +936,7 @@ queue_insert(Offline, MsgOrRef, #queue{max=-1, size=Size, queue=Queue} = Q, SId)
 queue_insert(_Offline, MsgOrRef, #queue{type=fifo, max=Max, size=Size, drop=Drop} = Q, SId)
   when Size >= Max ->
     on_message_drop_hook(SId, MsgOrRef, queue_full),
+    vmq_util:sample_log(1, "Message dropped for subscriber_id: ~p due to ~p", [SId, queue_full]),
```

Comment:
Why hard coding the threshold when we have the config? Same in [line 948](https://github.com/gojek/vernemq/pull/15/files#diff-a7a7879779f44e0b0c0c9a877b12aa328b114a0be29068480916ce95091941e3R948).